tron.bat

```diff
@@ -384,7 +385,7 @@ if /i %SCRIPT_VERSION% LSS %REPO_SCRIPT_VERSION% (
 			echo %TIME%   This copy of Tron will now self-destruct.
 			echo.
 			popd
-			pause
```

Comment:
This pause is purposefully here since we assume if someone is downloading Tron, it's not an unattended run and they are present to view the message. W/out pause the message flashes by too quickly and is pointless to include